packages/router/__tests__/router-test.ts

```diff
@@ -7004,6 +7004,45 @@ describe("a router", () => {
       }
     });
 
+    it("processes redirects with document reload if header is present (assign)", async () => {
+      let urls = ["/redirect"];
+
+      for (let url of urls) {
```

Comment:
We can remove the `urls` array here and in the test below and just test the singular URL - that was only needed in the above test to test different types of absolute URLs

Reply:
I purposely left it as similar as possible to the other test in case we wanted to add other urls to the test. But this is a perfectly reasonable simplification.